include/deal.II/lac/block_vector_base.h

```diff
@@ -142,10 +142,14 @@ namespace internal
        * work on. (Example: `std::next()`, which needs to know about a local
        * type named `difference_type`.)
        */
+#ifdef DEAL_II_HAVE_CXX20
+      using iterator_category = std::contiguous_iterator_tag;
```

Comment:
Is this true? I thought that these iterators were not contiguous because they span multiple vector components.

Reply:
Oh, that's actually quite a good question. One could ask that for `random_access_iterator_tag` too. I guess if we have a fixed number of blocks, then regardless of the size of the vector doing `it+n` is still O(1) effort. So at least conceptually, one can say that these iterators are random access. Do you agree?

You're right that the iterator is definitely not contiguous. I'll fix that once we have the question above settled.

Reply:
> guess if we have a fixed number of blocks, then regardless of the size of the vector doing it+n is still O(1) effort. So at least conceptually, one can say that these iterators are random access. Do you agree?
👍

`BlockVector` is like `std::deque` in that way.